TOC.md

```diff
@@ -138,6 +138,7 @@
     + [BR Use Cases](/br/backup-and-restore-use-cases.md)
   + TiDB Binlog
     + [Overview](/tidb-binlog/tidb-binlog-overview.md)
+    + [Quick Start](/tidb-binlog/get-started-with-tidb-binlog.md)
     + [Deploy](/tidb-binlog/deploy-tidb-binlog.md)
```

Comment:
Please also add `loader-overview.md` in TOC.

Reply:
It is already in TOC.